### ISSUE TITLE
zhTW only error

### ISSUE THREAD

**Issue by EKE00372** [erglo/mission-report-button-plus#9]
only happened when checking Camp Aylaag in zhTW client, will broke minimap button right click menu

`...onReportButtonPlus/MissionReportButtonPlus_Utils.lua:1923: bad argument #1 to 'strtrim' (string expected, got nil)`

**Comment by erglo** [comment]
Oh, I forgot about this. This is were I split the zone name from the subzone name to get a shorter display name. Those are usually separated by a `, ` (comma) character which is 8-bit of length. I forgot that most asian fonts use wide 16-bit characters.
I will fix this as soon as possible. Thank you for reporting this!

**Comment by erglo** [comment]
Fixed. Unfortunately I couldn't test it. Blizzard doesn't seem to allow entering an European server with a Chinese version of the game. I tried it with both traditional and simplified Chinese, but with no luck. Apparently, I'm not the only one (see this [post at Blizzard Support](https://eu.forums.blizzard.com/en/wow/t/chinese-language-on-eu-servers/286524)). It seems, that it isn't working with any non-European language.

But the error shouldn't return. There are 3 possible outcomes now:
1. it works, and displays the short name of the closest flight point, or
2. it displays the full name, including zone name (in this case I got the wrong character as separator), or
3. it returns an empty string.

Hopefully you keep me posted @EKE00372.
